examples/Divider.main.jsx

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Divider, Text } from '@itwin/itwinui-react';
+
+export default () => {
+  return (
+    <div className='demo-container'>
+      <Text variant='subheading'>Section 1</Text>
+      <Divider style={{ width: '500px' }} orientation='horizontal' />
```

Comment:
it would be good to not have to pass inline `style`s to `Divider` in these examples. it should automatically fill up the available width of the container (or height in the case of vertical)

Reply:
I have removed `width`, `height` and `orientation='horizontal'` since it is set by default already.